microcosm_logging/factories.py

```diff
@@ -113,21 +114,22 @@ def make_json_formatter(graph):
     Used for searchable aggregation.
 
     """
-    fields = {
-        # what was logged?
-        "loggerName": "%(name)s",
-        "levelName": "%(levelname)s",
-        "message": "%(message)s",
-        # when was it logged?
-        "ascTime": "%(asctime)s",
-        # where in the code was it logged?
-        "fileName": "%(filename)s",
-        "functionName": "%(funcName)s",
-        "lineNo": "%(lineno)d",
-    }
+    fields = [
+        'asctime',
+        'levelname',
+        'name',
+        'filename',
+        'funcName',
+        'lineno',
+        'message',
+    ]
+
+    log_format = lambda x: ['%({0:s})'.format(i) for i in x]
+    custom_format = ' '.join(log_format(fields))
+
     return {
-        "format": dumps(fields),
-        "datefmt": "",
+        '()': 'pythonjsonlogger.jsonlogger.JsonFormatter',
```

Comment:
Maybe I'm not understanding how this line works but should we call `jsonlogger` since we imported it?